k8s/overlays/staging/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
We probably will need a name for what the users think of staging that won't confuse us - preview or testing perhaps?

Reply:
have changed naming to testing... because you are right we dont have a specific staging k8s cluster